samples/word/change_the_print_orientation/cs/Program.cs

```diff
@@ -37,7 +38,7 @@ static void SetPrintOrientation(string fileName, string no)
         {
             bool pageOrientationChanged = false;
 
-            PageSize pgSz = (sectPr.Descendants<PageSize>().FirstOrDefault()) ?? throw new System.NullReferenceException("There are no PageSize elements in the section.");
+            PageSize pgSz = (sectPr.Descendants<PageSize>().FirstOrDefault()) ?? throw new ArgumentNullException("There are no PageSize elements in the section.");
```

Comment:
Change this to `.First()` instead of throwing exceptions - it'll ensure there is at least one